docs/developer-guide/contributors-quickstart.md

```diff
@@ -49,6 +49,28 @@ make install-codegen-tools-local
 
 Install Go with a version equal to or greater than the version listed in `go.mod` (verify go version with `go version`). 
 
+### Install Goreman
+
+1. Add these lines to your ~/.bashrc:
+   ```bash
+    export GOPATH=$HOME/go
+    export PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+    export PATH=$GOPATH/bin:$PATH
+   ```
+   Then reload the shell config:
+   ```bash
+   source ~/.bashrc
+   ```
+
+2. Install goreman:
+   ```bash
+   go install github.com/mattn/goreman@latest
+   ```
+
+3. Verify installation:
+   ```bash
+   goreman -h
+   ```
```

Comment:
Rather than writing all of this, let's simply provide the link to goreman installation - https://github.com/mattn/goreman?tab=readme-ov-file#getting-started